workflows/phylorthology.nf

```diff
@@ -116,18 +488,19 @@ workflow PHYLORTHOLOGY {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-workflow.onComplete {
-    if (params.email || params.email_on_fail) {
-        NfcoreTemplate.email(workflow, params, summary_params, projectDir, log, multiqc_report)
-    }
-    NfcoreTemplate.summary(workflow, params, log)
-    if (params.hook_url) {
-        NfcoreTemplate.adaptivecard(workflow, params, summary_params, projectDir, log)
-    }
-}
+//workflow.onComplete {
```

Comment:
eventually you might want to add this back in when it's launched on tower because then there is nice email notification integration i believe

Reply:
Okay - I think I deleted/commented this all out early on before I knew what anything was doing, lol.